.github/workflows/ci.yml

```
@@ -48,7 +48,7 @@ jobs:
           name: code-coverage-ubuntu-latest-20.x
           path: coverage/
       # Note: sonarcloud doesn't publish a major version tag, which is best practice for github actions.
-      - uses: SonarSource/sonarcloud-github-action@v2.2.0
+      - uses: SonarSource/sonarcloud-github-action@v2.3.0
```

```suggestion
      - uses: SonarSource/sonarcloud-github-action@v2
```

Better to use the `v2` tag formulation